api/provider/aws/logging.go

```diff
@@ -16,6 +16,73 @@ const (
 	MAX_DESCRIBE_STREAMS_COUNT = 1000
```

Comment:
I realize this isn't part of your change but can you please update this to use `mixedCasing` if it isn't used outside of this package or `MixedCasing` if it is.